### PR TITLE
2.11 version of the spec is on a branch

### PR DESCRIPTION
**Pull request by avade**
As per the proposed release process, we will make a branch when a version of the spec is released.

### CODE REVIEW

**Comment by angarg12** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
-[The Open Service Broker API specification](spec.md)
+## Latest Release: v2.11
+[Read the specification](https://github.com/openservicebrokerapi/servicebroker/tree/v2.11)
 
 [Release Notes for past versions](release-notes.md)
 
+## Working Changes
```
Comment (duglin) [inline README.md]: I'm assuming this new section is for changes that have been through the "implementation" phase and are accepted to be part of the next rev of the spec but we just haven't released a new version - correct?  IOW, this isn't for changes that are still under "implementation review".

Comment (avade) [inline README.md]: Correct - I see this as anything that has been merged into master ready for the next release. However, I wanted to call out that we still reserve the right to change things before the official release.

Comment (duglin) [inline README.md]: ok thanks

**Comment by duglin** [comment]
LGTM

**Comment by avade** [comment]
@pmorie your LGTM wasn't picked up as it was a review comment, not a regular comment.

**Comment by avade** [comment]
@shalako @zrob @vaikas-google @fmui - can y'all take a look at this PR. One more approval is needed.

**Comment by zrob** [comment]
LGTM

**Pull request merged**